### PR TITLE
4.6.0 New Data Proxy error codes

### PR DESCRIPTION
**Pull request by zachtil**
## Describe this PR

Add new Data Proxy error codes to Error reference page.

## Changes

New error codes:

* P5011
* P5012
* P5013
* P5014
* P5015

## What issue does this fix?

#3986

### CODE REVIEW

File: content/800-data-platform/300-data-proxy.mdx [hunk 1]
```diff
@@ -156,24 +156,28 @@ You are now ready to try the Data Proxy. Go ahead and start your app!
 
 The Data Proxy does not yet support the following preview features:
 
-- [Interactive Transactions](/concepts/components/prisma-client/transactions#interactive-transactions-in-preview)
+
```
Comment (aqrln) [inline content/800-data-platform/300-data-proxy.mdx]:
```suggestion
```

**Review by aqrln — APPROVED** [review]
Looks great!

File: content/800-data-platform/300-data-proxy.mdx (hunk 1 above)
Comment (aqrln) [reply content/800-data-platform/300-data-proxy.mdx]: (double empty line can be replaced with just one)

Comment (zachtil) [reply content/800-data-platform/300-data-proxy.mdx]: Good catch. It's removed now. Thanks, @aqrln

**Pull request merged**